wallet/createtx_test.go

```diff
@@ -0,0 +1,253 @@
+package wallet
```

Comment:
Where did this file come from?
It is not in dcrwallet master as far as I can tell.  And it doesn't compile.